### PR TITLE
First draft of "Connected FT"

### PR DESCRIPTION
**Pull request by seanmtracey**
Image-less atm, will add once copy has been critiqued and agreed.

### CODE REVIEW

File: _posts/2017-07-03-connected-ft.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+excerpt: >
+  Connecting personal devices to each other with Web Push and FT.com
+---
+
+<p>Imagine being able to send any piece of FT content from one of your devices to another with the single press of a button...</p>
+
+<p>Our customers consume FT content across countless devices. From the 27" 4K iMac to sub-$20 Android devices, to the aged windows boxes collecting dust in the corner, the FT can be read almost anywhere, and we strive for a great experience across as many devices as possible.</p>
+
+<p>Despite the FT being available on almost any web-capable device, it might not always be the best place for you to view that content. Perhaps you're reading a gripping piece on your laptop in the morning, but you have to get to work... your smartphone is probably the best way for you to conusme that must-read article on the move. But how do you get it there? Well, there are a bunch of ways that you can get an article from point A to point B, but they're convoluted. To pick a favourite example: you could email it to yourself, but then you have an email which isn't really an email, it's a bookmark, and that's not hat your inbox is for.</p>
+
```
Comment (railsagainstignorance) [inline _posts/2017-07-03-connected-ft.html]: "that's not hat"

Comment (seanmtracey) [reply _posts/2017-07-03-connected-ft.html]: `hat` -> `what`

File: _posts/2017-07-03-connected-ft.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+<p>As part of a 20% time project, we've built 'Connected FT', a small web service + progressive web app that makes it possible to send articles between your devices with a single button press.</p>
+
+<h2>How does it work?</h2>
+
+<p>Connected FT uses the new Web Push APIs to connect each of your personal devices to your FT account. With push notifications enabled, we're able to send you any content you elect to receive on that device.</p>
+
+<p>To enable Connected FT on a device, you simply register it with our experiental PWA</p>
+
```
Comment (railsagainstignorance) [inline _posts/2017-07-03-connected-ft.html]: experiental PWA

File: _posts/2017-07-03-connected-ft.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+<p>Once registration is complete, a new sharing button will appear alongside our default sharing options. To send an article to another device, you simply click the button, select your desired device from the list that appears.</p>
+
+<p>Voila, as soon as you're ready for it, the content you want is waiting right there for you.</p>
+
+<h2>Technical Limitations</h2>
+
+<p>As with most shiny new Web APIs, the Web Push API is not supported absolutely everywhere - most notably on iOS (unsurprisingly). At current, the best experience for this technology is on Android devices with a modern web browser (Chrome/Firefox/Samsung Internet)</p>
+
+<h2>How else coudld we use this?</h2>
+
```
Comment (railsagainstignorance) [inline _posts/2017-07-03-connected-ft.html]: coudld

File: _posts/2017-07-03-connected-ft.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+<p>While it's very convenient to be able to send an article from one device to another, it's not neccessarily a use case that absolutely everybody has. We imagine that Connected FT might be best used in circumstances where a screen is not immediately available to read an article on. For example, one could be listening to an FT article on their shiny new Google Home/Amazon Alexa/Shiny voice vase and then decide that they'd prefer to read the article on their phone (perhaps they have to run out of the door). With a simple voice command, such as "Hey Vase, send this article to my phone", it can be there in seconds. No muss, no fuss. </p>
+
```
Comment (railsagainstignorance) [inline _posts/2017-07-03-connected-ft.html]: too many shinys

also, no "muss"?

File: _posts/2017-07-03-connected-ft.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+<p>Our customers consume FT content across countless devices. From the 27" 4K iMac to sub-$20 Android devices, to the aged windows boxes collecting dust in the corner, the FT can be read almost anywhere, and we strive for a great experience across as many devices as possible.</p>
+
+<p>Despite the FT being available on almost any web-capable device, it might not always be the best place for you to view that content. Perhaps you're reading a gripping piece on your laptop in the morning, but you have to get to work... your smartphone is probably the best way for you to conusme that must-read article on the move. But how do you get it there? Well, there are a bunch of ways that you can get an article from point A to point B, but they're convoluted. To pick a favourite example: you could email it to yourself, but then you have an email which isn't really an email, it's a bookmark, and that's not what your inbox is for.</p>
+
+<p>As part of a 20% time project, we've built 'Connected FT', a small web service + progressive web app that makes it possible to send articles between your devices with a single button press.</p>
+
```
Comment (railsagainstignorance) [inline _posts/2017-07-03-connected-ft.html]: --> "As a 20% time project", dropping "part of"

File: _posts/2017-07-03-connected-ft.html (hunk 4 above)
Comment (seanmtracey) [reply _posts/2017-07-03-connected-ft.html]: http://www.ldoceonline.com/dictionary/no-muss-no-fuss

**Pull request merged**